app/content/app.css

```diff
@@ -1,125 +1,125 @@
-body {
-  font-family: Oxygen, Helvetica, Arial, sans-serif; }
-
-h1, h2, h3, h4, h5 {
-font-family: Merriweather;
-font-weight: 300;
-color: #FE5F55; }
-
-.container {
-  width: 100% !important;
-  padding-left: 0 !important;
-  padding-right: 0 !important; }
-
-.row {
-  margin-left: 0 !important;
-  margin-right: 0 !important; }
-
-.col-xs-12 {
-  padding-left: 0 !important;
-  padding-right: 0 !important; }
-
-header .well {
-  padding-top: 5px;
-  padding-bottom: 5px; }
-  header .well .user-links {
-    float: right;
-    color: #ccc; }
-    header .well .user-links a {
-      cursor: pointer; }
-
-a:link, a:visited, a:hover, a:active {
-  color: #FE5F55; }
-
-i.fa-trash-o {
-  cursor: pointer;
-  color: #ccc;
-  vertical-align: middle;
-  margin-left: 1em; }
-  i.fa-trash-o:hover {
-    color: #999; }
-
-.wysihtml5-toolbar .btn-default {
-  color: #999; }
-
-input#note_title {
-  border: none;
-  font-size: 200%;
-  font-family: Merriweather;
-  color: #FE5F55;
-  font-weight: 300;
-  width: 100%; }
-  input#note_title:focus {
-    outline: none; }
-
-ul#notes {
-  overflow-y: scroll;
-  height: calc(100vh - 110px);
-  list-style: none;
-  margin-top: 1em;
-  padding: 0;
-  width: 100%;
-  color: #999; }
-  ul#notes li {
-    border-top: 1px solid #ddd;
-    height: 100px;
-    font-size: 90%;
-    cursor: pointer;
-    overflow: hidden; }
-    ul#notes li:first-of-type {
-      border-top: none; }
-    ul#notes li:hover {
-      color: white;
-      background-color: #FE5F55; }
-      ul#notes li:hover .note-title {
-        color: white; }
-    ul#notes li .note-title {
-      color: black;
-      font-family: Merriweather;
-      font-size: 120%;
-      font-weight: 300; }
-    ul#notes li .note-body {
-      height: 54px;
-      overflow: hidden; }
-    ul#notes li a.note {
-      display: block;
-      height: 100px;
-      padding: 1em;
-      text-decoration: none; }
-    ul#notes li a.note:hover {
-      color: white; }
-
-#new_user input[type="text"],
-#new_user input[type="password"] {
-  width: 300px;
-  font-size: 120%;
-  border-radius: 4px;
-  border: 1px solid #ccc;
-  padding: 0.25em; }
-
-#welcome h1 {
-  color: #FE5F55; }
-#welcome main {
-  margin: 0 auto;
-  width: 360px; }
-
-span.login {
-  margin-left: 1em; }
-
-.ta-editor {
-  margin-top: 0.5em; }
-
-body .alert.alert-success {
-  padding: 0;
-  margin: 0;
-  display: inline-block;
-  border: none;
-  position: absolute;
-  right: 1rem;
-  bottom: 0;
-  color: #d3d3d3;
-  background-color: inherit;
-}
-body .alert-danger {
-  margin-top: 1rem;
-}
+body {
+  font-family: Oxygen, Helvetica, Arial, sans-serif; }
+
+h1, h2, h3, h4, h5 {
+font-family: Merriweather;
+font-weight: 300;
+color: #FE5F55; }
+
+.container {
+  width: 100% !important;
+  padding-left: 0 !important;
+  padding-right: 0 !important; }
+
+.row {
+  margin-left: 0 !important;
+  margin-right: 0 !important; }
+
+.col-xs-12 {
+  padding-left: 0 !important;
+  padding-right: 0 !important; }
+

```

Comment:
We should try to avoid using `!important` as much as we can. `!important` overrides how styles apply so if we want to use `.container`, `.row`, or `.col-xs-12` later on we won't be able to override these styles specifically.

Alternatively, we would try to scope the styles a little further. Something like this should work:
```
.container.wrap {
  width: 100%;
  padding-left: 0;
  padding-right: 0;
}

.container .row {
  margin-left: 0;
  margin-right: 0;
}

.row .col-xs-12 {
  padding-left: 0;
  padding-right: 0;
}
```